### ISSUE TITLE
Crash when creating a poll with special chars

### ISSUE THREAD

**Issue by chuinul** [matterpoll/matterpoll#319]
#### Summary
Special characters in poll options crashes desktop & web clients.

#### Steps to reproduce
Just create a poll with an option containing a special character (I couldn’t tell what kind of character triggers the bug, I did not encounter any issue with common characters like 'é'):
`/poll "Do you need help?" "No, thanks" "🚒"`

#### Expected behavior
The poll should be displayed with special chars properly rendered.

#### Observed behavior (that appears unintentional)
A white bank screen. Need to restart the client or close the tab of the browser.

#### Server configuration
**Operating system:** Linux Mint 20 Cinnamon

**Mattermost version:** 5.23.0

**Matterpoll version:** 1.3.0

**Updated from an older Matterpoll version or fresh install:** fresh install

**Comment by hanzei** [comment]
I tried to repo the bug with a couple of Russian and Arabic character but couldn't. Could you please try to research what special character caused the crash?

**Comment by kaakaa** [comment]
With ExperimentalUI settings, I can repro this issue. So if you set `ExperimentalUI` to false, this issue is resolved.

---

The cause may be that plugin utility function `PostUtils.formatText` with `{markdown: false}` option seems not to be enable to handle unicode emojis.

Simple repro example is that `PostUtils.formatText('🚒', { markdown: false });` will be crashed.

```
text_formatting.tsx:854 Uncaught TypeError: Cannot read property 'hasUnicode' of undefined
    at text_formatting.tsx:854
    at RegExp.[Symbol.replace] (<anonymous>)
    at String.replace (<anonymous>)
    at text_formatting.tsx:835
    at g (text_formatting.tsx:281)
    at Object._ [as formatText] (text_formatting.tsx:230)
    at s.initialize (com.mattermost.plugin-starter-template_63e7ac464064c09d_bundle.js:1)
    at index.js:135
    at HTMLScriptElement.i.onload (index.js:99)
```

I'll create an issue in mattermost-server.

**Comment by chuinul** [comment]
Yes correct @kaakaa, disabling `ExperimentalUI` fixes the issue. Thanks for having reporting a bug in mattermost-server. Shall I close that issue?

**Comment by kaakaa** [comment]
Great 👍  If you are fine, I want to stay this issue open. I'll close this issue when the cause is resolved or this limitation is written in document. 

And thank you for reporting @chuinul .

**Comment by hanzei** [comment]
This has been fix in Mattermost v5.28, which will ship on the 16th of October.
